src/services/blocks/BlocksService.ts

```diff
@@ -444,18 +472,26 @@ export class BlocksService extends AbstractService {
 	 * @param api ApiPromise
 	 * @param parentHash Hash of the parent block
 	 * @param block Block which the extrinsic is from
+	 * @param block
 	 */
 	private async createCalcFee(
 		api: ApiPromise,
 		parentHash: Hash,
-		block: Block
+		block: Block,
+		that: That
 	): Promise<ICalcFee> {
 		const perByte = api.consts.transactionPayment?.transactionByteFee;
 		const extrinsicBaseWeightExists =
 			api.consts.system.extrinsicBaseWeight ||
 			api.consts.system.blockWeights.perClass.normal.baseExtrinsic;
 
-		let calcFee, specName, specVersion, decorated, runtimeDoesNotMatch;
+		/**
+		 * Chains where extrinsicBaseweight data is hardcoded in Sidecar
+		 * to avoid using expandMetadata to grab it
+		 */
+		const hardcodedChains = ['polkadot', 'kusama'];
+
+		let calcFee, specName, specVersion, decorated;
```

Comment:
Instead of returning decorated this should just muatate the `metaConstsCache`.

Pseudo logic is 
```javascript
metaConstsCache[runtimeVersion] ||= this.getDecorateAndExtractWeight(blockNumber)
```